### ISSUE TITLE
( ! ) Deprecated: Function openssl_free_key() is deprecated in /ondrakoupil/csob-eapi-paygate/src/Crypto.php on line 49

### ISSUE THREAD

**Issue by Vitexus** [ondrakoupil/csob#33]
Update Crypto for **PHP Version 8.0.3**

**Comment by Vitexus** [comment]
The [manual say](https://www.php.net/manual/en/function.openssl-free-key.php):

8.0.0 | This function is now deprecated as it doesn't have an effect anymore.

Two function calls simply removed.

**Comment by Vitexus** [comment]
Maybe better solution now is condition: 

```
                if(version_compare(PHP_VERSION, '8.0.0', '<'  ))
```

**Comment by ondrakoupil** [comment]
Thanks. The issue is fixed in v1.8.4 in 2c175ad
